oak_runtime/src/node/http/tests.rs

```diff
@@ -45,6 +46,58 @@ impl crate::node::Node for OakNodeForTest {
         let invocation_receiver = Receiver::<HttpInvocation>::new(ReadHandle { handle });
 
         while let Ok(invocation) = invocation_receiver.receive(&runtime) {
+            // Create a request-specific EchoNode, with a privilege that allows the node to
+            // declassify the request.
+            let privilege = NodePrivilege {
+                can_declassify_confidentiality_tags: hashset! {tls_endpoint_tag("localhost")},
+                can_endorse_integrity_tags: hashset! {},
+            };
```

Comment:
The fact that we can create nodes with arbitrary privilege made me realise that we should probably make sure that this enforced by the runtime, I created https://github.com/project-oak/oak/issues/1636 to track.

Reply:
IIUC, only public nodes can create new nodes. With this restriction in privilege, we will never be able to create any node with a privilege unless the initial public node has the top privilege. Is this the assumption?